src/components/Articles/ArticleCard.tsx

```diff
@@ -42,7 +42,7 @@ export default function PostCard({ post }: PostCardProps) {
         <LikeButton favoritesCount={favoritesCount} />
       </div>
       <div className={styles.contentWrapper}>
-        <Link href={`${ROUTES.articleDetail}${slug}`}>
+        <Link href={`${ROUTES.ARTICLEDETAIL(slug).href}`}>
```

Comment:
요기도 이렇게 되면 좋을 것 같아요~

```suggestion
        <Link href={`${ROUTES.ARTICLE_DETAIL(slug).href}`}>
```